.github/workflows/ci-build.yml

```diff
@@ -52,7 +52,6 @@ jobs:
           echo "dev.docs.pyansys.com" > doc/build/html/CNAME
 
       - name: Deploy to gh-pages on main
-        if: github.ref == 'refs/heads/main'
```

Comment:
Web documentation always shows the latest version. I remember I told you in our meeting those were published only when doing a release, but now I see this is not the case.

This line here guarantees that only changes merged into the main are deployed. We need to keep this line, otherwise, we would have docs rendered every time a developer pushes a new commit to a PR.